pkg/user/delivery/http/user_proxy.go

```diff
@@ -87,6 +102,42 @@ func (p *Proxy) CreateToken(user entities.User) (string, error) {
 	return tr.Token, nil
 }
 
+// CreateAppToken creates a valid token for the application
+func (p *Proxy) CreateAppToken(email, userToken string) (string, error) {
+	body, err := json.Marshal(&AppTokenRequest{Email: email, Type: 2, Duration: 3153600000})
```

Comment:
What about creating a constant to store that duration value.

Reply:
Please see [this comment](https://github.com/CESARBR/knot-babeltower/pull/78#issuecomment-636193722) to understand the -1! :grimacing:

Reply:
Agreed. Considering our time and needs, it's better to expose the feature in a similar way (but we don't necessarily need to expose as it is always).

Reply:
I add a new commit to handle the duration time. Please check if it is ok! :smiley: